ft_databrowser.m

```diff
@@ -49,6 +49,8 @@
 %   cfg.visible                 = string, 'on' or 'off' whether figure will be visible (default = 'on')
 %   cfg.position                = location and size of the figure, specified as a vector of the form [left bottom width height]
 %   cfg.renderer                = string, 'opengl', 'zbuffer', 'painters', see MATLAB Figure Properties. If this function crashes, you should try 'painters'.
+%   cfg.colormap                = string, or Nx3 matrix, colormap for the
```

Comment:
can you put the help on a single line (it is not that long) and can you make it consistent with other ft functions?

```
mac036-lan> cd fieldtrip
mac036-lan> grep "%   cfg.colormap" *.m
ft_multiplotTFR.m:%   cfg.colormap         = any sized colormap, see COLORMAP
ft_singleplotTFR.m:%   cfg.colormap       = any sized colormap, see COLORMAP
ft_topoplotER.m:%   cfg.colormap           = any sized colormap, see COLORMAP
ft_topoplotIC.m:%   cfg.colormap           = any sized colormap, see COLORMAP
ft_topoplotTFR.m:%   cfg.colormap           = any sized colormap, see COLORMAP
```